config.w32

```diff
@@ -40,24 +40,26 @@ if (PHP_MONGODB != "no") {
   if (CHECK_LIB("ssleay32.lib", "mongodb", PHP_MONGODB) &&
       CHECK_LIB("libeay32.lib", "mongodb", PHP_MONGODB) &&
       CHECK_HEADER_ADD_INCLUDE("openssl/ssl.h", "CFLAGS_MONGOC")) {
-    AC_DEFINE("MONGOC_ENABLE_OPENSSL", 1);
-    AC_DEFINE("MONGOC_ENABLE_LIBCRYPTO", 1);
+    AC_DEFINE("MONGOC_ENABLE_SSL_OPENSSL", 1);
+    AC_DEFINE("MONGOC_ENABLE_CRYPTO_LIBCRYPTO", 1);
     AC_DEFINE("MONGOC_ENABLE_SSL", 1);
     AC_DEFINE("MONGOC_ENABLE_CRYPTO", 1);
+    AC_DEFINE("MONGOC_ENABLE_CRYPTO_SYSTEM_PROFILE", 1);
```

Comment:
I noted that `CMakeLists.txt` (for Windows) in libmongoc always sets this to 1, so I've opted to do the same here.